### DISCUSSION TITLE
private elevator

### DISCUSSION THREAD

**Discussion by Leahcimkrob** [Zrips/CMI#8186] [Feature Requests]
Is it possible to expand the elevator so that you can also set it to private, i.e. So that you can create elevators on your plot that you can only use as the creator of the elevator?

**Comment by CloudeLecaw** [comment]
Currently its a feature that is just seen by the sign itself.
it has no creator or other database like warps or homes does

**Comment by Leahcimkrob** [comment]
I meant it more like that on the sign in the 3rd line [private] comes in and then only the plot owner (or region owner) and those trusted on the plot can use the sign.

**Comment by CloudeLecaw** [comment]
well that cant be done.
but may it can be made with a check if you have access to build or not.. :)

**Comment by Leahcimkrob** [comment]
That would be enough.

**Comment by Leahcimkrob** [comment]
CloudeLecaw, were you able to do something about that?

**Reply by CloudeLecaw** [reply]
nothing like this is added to the cmi elevators. so technically no from cmi site.

On our server our region claim plugin has a region sign flag that players can set to false. its for interact with signs on their regions. The flag is set to true by default. when set to false it blocks strangers from interact with all types of signs (also when its a shop or anything else)
